src/Tribe/Repositories/Attendee/RSVP.php

```diff
@@ -14,6 +18,37 @@ class Tribe__Tickets__Repositories__Attendee__RSVP extends Tribe__Tickets__Atten
 	 */
 	protected $key_name = 'rsvp';
 
+	/**
+	 * {@inheritdoc}
+	 */
+	public function __construct() {
+		parent::__construct();
+
+		$this->attendee_provider = tribe( 'tickets.rsvp' );
+
+		$this->create_args['post_type'] = $this->attendee_provider->attendee_object;
+
+		// Use a regular variable so we can get constants from it PHP <7.0
+		$attendee_provider = $this->attendee_provider;
+
+		// Add object specific aliases.
+		$this->update_fields_aliases = array_merge( $this->update_fields_aliases, [
+			'ticket_id'       => $attendee_provider::ATTENDEE_PRODUCT_KEY,
+			'event_id'        => $attendee_provider::ATTENDEE_EVENT_KEY,
+			'post_id'         => $attendee_provider::ATTENDEE_EVENT_KEY,
+			'security_code'   => $attendee_provider->security_code,
+			'order_id'        => $attendee_provider->order_key,
+			'optout'          => $attendee_provider::ATTENDEE_OPTOUT_KEY,
+			'user_id'         => $attendee_provider->attendee_user_id,
+			'price_paid'      => '_paid_price',
+			'full_name'       => $attendee_provider->full_name,
+			'email'           => $attendee_provider->email,
+			'attendee_status' => $attendee_provider::ATTENDEE_RSVP_KEY,
+		] );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: bcaae23013eb13629603356b046b2936</sub>